sentry-spring-boot/src/main/java/io/sentry/spring/boot/SentryAutoConfiguration.java

```diff
@@ -164,7 +164,7 @@ static class OpenTelemetryLinkErrorEventProcessorConfiguration {
     }
 
     @Configuration(proxyBeanMethods = false)
-    @Import(SentryGraphqlConfiguration.class)
```

Comment:
Can the `SentryGraphqlConfiguration` class be deleted, or is it still of use?

Reply:
Still in use by people not using Spring Boot, so keeping it, see above.